docs/docsite/rst/network/getting_started/intermediate_concepts.rst

```diff
@@ -0,0 +1,79 @@
+
+A Typical Ansible Filetree
+================================================================================
+
+Ansible expects to find certain files in certain places. As you expand your inventory and create and run more network playbooks, keep your files organized in your working Ansible project directory like this:
+
+.. code-block:: console
+
+   .
+   ├── backup
+   │   ├── vyos.example.net_config.2018-02-08@11:10:15
+   │   ├── vyos.example.net_config.2018-02-12@08:22:41
+   ├── first_playbook.yml
+   ├── inventory
+   ├── group_vars
+   │   ├── vyos.yml
+   │   └── eos.yml
+   ├── roles
+   │   ├── static_route
+   │   └── system
+   ├── second_playbook.yml
+   └── third_playbook.yml
+
+The ``backup`` directory and the files in it get created when you run modules like ``vyos_config`` with the ``backup: yes`` parameter.
+
+
+Tracking Changes to Inventory and Playbooks: Source Control with Git
+================================================================================
+
+As you expand your inventory, roles and playbooks, you should place your Ansible projects under source control. We recommend ``git`` for source control. ``git`` provides an audit trail, letting you you track changes, roll back mistakes, view history and share the workload of managing, maintaining and expanding your Ansible ecosystem. There are plenty of tutorials and guides to using ``git`` available.
+
+Resources and Next Steps
+================================================================================
+
+Text:
+--------
+
+Network Automation on the `Link Ansible website <http://ansible.com/overview/networking>`
+Ansible Network `Link Blog posts <http://ansible.com/blog/topic/networks>`
+
+Video:
+--------
+
+All the `Link AnsibleFests <https://www.ansible.com/resources/videos> and `Link AnsibleAutomates <https://www.ansible.com/resources/webinars-training>` are recorded and include many Network related topics.
+You can filter to only show Network topics using the Filter by Category option on the right hand side
+For the schedule the events please see the Ansible Events page. 
+
+GitHub Repos:
+----------------
+
+The Network-Automation GitHub project is a open community for all things network automation. Here there will be various examples, demonstrations and GitHub repos dedicated exclusively for automation networking platforms. 
+Lightbulb
```

Comment:
Remove "lightbullb, example playbooks, with other repos" they were reminders.

Let's get this live then Sean etc can suggest updates